montepy/data_inputs/tally_segment.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024, Battelle Energy Alliance, LLC All Rights Reserved.
+import montepy
+from montepy.data_inputs.data_input import DataInputAbstract
+from montepy.input_parser.tally_parser import TallyParser
+
+
+class TallySegment(DataInputAbstract):
+    """ """
+
+    _parser = TallyParser()
```

Comment:
`TallySegment` might need its own parser (unless `TallyParser` has an option to also support `C`)

Reply:
I agree.